core/objects.go

```diff
@@ -772,81 +771,39 @@ type SignedCertificateTimestamp struct {
 	LockCol int64
 }
 
-// RPCSignedCertificateTimestamp is a wrapper around SignedCertificateTimestamp
-// so that it can be passed through the RPC layer properly. Without this wrapper
-// the UnmarshalJSON method below will be used when marshaling/unmarshaling the
-// object, which is not what we want as it is not symmetrical (as it is intended
-// to unmarshal a rawSignedCertificateTimestamp into a SignedCertificateTimestamp)
-type RPCSignedCertificateTimestamp SignedCertificateTimestamp
-
-type rawSignedCertificateTimestamp struct {
-	Version    uint8  `json:"sct_version"`
-	LogID      string `json:"id"`
-	Timestamp  uint64 `json:"timestamp"`
-	Signature  string `json:"signature"`
-	Extensions string `json:"extensions"`
-}
-
-// UnmarshalJSON parses the add-chain response from a CT log. It fills all of
-// the fields in the SignedCertificateTimestamp struct except for ID and
-// CertificateSerial, which are used for local recordkeeping in the Boulder DB.
-func (sct *SignedCertificateTimestamp) UnmarshalJSON(data []byte) error {
-	var err error
-	var rawSCT rawSignedCertificateTimestamp
-	if err = json.Unmarshal(data, &rawSCT); err != nil {
-		return fmt.Errorf("Failed to unmarshal SCT receipt, %s", err)
-	}
-	sct.LogID = rawSCT.LogID
+// InternalToSCT converts a internal SCT object to a google SCT object
+func InternalToSCT(iSCT SignedCertificateTimestamp) (*ct.SignedCertificateTimestamp, error) {
```

Comment:
This doesn't appear to be used anywhere. Delete it?

And its counterpart, SCTToInternal, appears to be used only in Publisher. Move it there? And add a test?